### DISCUSSION TITLE
[2021.09.07] 프린터

### DISCUSSION THREAD

**Discussion by hyk339** [pyunji/Hyundai-ITnE-Team_QuadCore#113] [Algorithm]
[프린터](https://programmers.co.kr/learn/courses/30/lessons/42587)

**Comment by hyk339** [comment]
[프린터 solved](https://github.com/hyk339/Hyundai-ITnE-Team_QuadCore/blob/master/khy/programmers/PR_42587.java)

**Comment by chaeyeon4u** [comment]
[프린터](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/ycy/homework/Java/PR_42587.java)

**Comment by pyunji** [comment]
[프린터](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/pyunji/pyunji/PR/PR_42587.java)

**Comment by leechang4** [comment]
[프린터 solved](https://github.com/leechang4/Hyundai-ITnE-Team_QuadCore/blob/master/lch/Programmers/PR_42587.java)
